### PR TITLE
fix(tangle-dapp): Fix links and site description copy

### LINKED ISSUE #2343: [TASK] Fix links to docs and check all other links
- [x] `Learn More` link is broken on Nomination page
image
- [x] `Learn More` link is broken on Account page
image

### PR DESCRIPTION
**Pull request by yurixander**
## Summary of changes

_Provide a detailed description of proposed changes._

- Fixed link to docs.
- Updated site description text shown on account page.
- Removed `simple-import-sort` eslint plugin, since it is causing annoyances and doesn't really provide any benefit.

### Proposed area of change

_Put an `x` in the boxes that apply._

- [ ] `apps/bridge-dapp`
- [ ] `apps/hubble-stats`
- [ ] `apps/stats-dapp`
- [x] `apps/tangle-dapp`
- [ ] `apps/testnet-leaderboard`
- [ ] `apps/faucet`
- [ ] `apps/zk-explorer`
- [ ] `libs/webb-ui-components`

### Reference issue to close (if applicable)

_Specify any issues that can be closed from these changes (e.g. `Closes #233`)._

- Closes #2343

### Screen Recording

_If possible provide a screen recording of proposed change._

---

### Code Checklist

_Please be sure to add .stories documentation if any additions are made to `libs/webb-ui-components`._

- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

### CODE REVIEW

File: apps/tangle-dapp/.eslintrc.json [hunk 1]
```diff
@@ -6,7 +6,7 @@
     "../../.eslintrc.json",
     "eslint:recommended"
   ],
-  "plugins": ["unused-imports", "simple-import-sort"],
```
Comment (AtelyPham) [inline apps/tangle-dapp/.eslintrc.json:9]: Is there an issue with this ESLint plugin that makes you want to remove it? I think it helps us organize our imports.

File: apps/tangle-dapp/containers/ValidatorTablesContainer/ValidatorTablesContainer.tsx [hunk 2]
```diff
@@ -51,7 +53,7 @@ const ValidatorTablesContainer = () => {
             "
             buttonText="Learn More"
             buttonProps={{
-              onClick: () => window.open(TANGLE_STAKING_URL, '_blank'),
```
Comment (AtelyPham) [inline apps/tangle-dapp/containers/ValidatorTablesContainer/ValidatorTablesContainer.tsx:54]: We can use href and target here to improve accessibility. The `Button` component already handles rendering either `<button />` or `<a />` based on the props.

**Review by AtelyPham — COMMENTED** [review]
Left some comments.

File: apps/tangle-dapp/.eslintrc.json (hunk 1 above)
Comment (yurixander) [reply apps/tangle-dapp/.eslintrc.json:9]: While making changes it keeps erroring and I have to keep constantly fixing it, and on this specific PR it wasn't reporting the error for some reason 👀 It does help organize the imports by sorting them, but I would say it's a bit buggy 😅

Comment (AtelyPham) [reply apps/tangle-dapp/.eslintrc.json:9]: You can use this command on VSCode to fix the issue automatically.

CleanShot 2024-05-30 at 13 25 42

Comment (yurixander) [reply apps/tangle-dapp/.eslintrc.json:9]: Yes, that's what I currently use. However, the problem is that I have to be doing that constantly as I make changes everywhere 👀

**Review by AtelyPham — APPROVED** [review]

**Pull request merged**